frontend/src/EmailValidationPage.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from 'react'
+import { Trans } from '@lingui/macro'
+import { Heading, Text, Stack, Icon, Divider } from '@chakra-ui/core'
+import { useParams } from 'react-router-dom'
+import { useMutation } from '@apollo/client'
+import { VERIFY_ACCOUNT } from './graphql/mutations'
+import { LoadingMessage } from './LoadingMessage'
+
+export default function EmailValidationPage() {
+  const { verifyToken } = useParams()
+  const [success, setSuccess] = useState(false)
+  let errorMessage = ''
+
+  const [verifyAccount, { loading }] = useMutation(VERIFY_ACCOUNT, {
+    onError(error) {
+      errorMessage = error.message
+    },
+    onCompleted({ verifyAccount }) {
+      if (verifyAccount.result.__typename === 'VerifyAccountResult')
+        setSuccess(true)
+      else if (verifyAccount.result.__typename === 'VerifyAccountError')
+        errorMessage = verifyAccount.result.description
+      else console.log('Incorrect verifyAccount.result typename.')
+    },
+  })
+
+  useEffect(() => {
+    verifyAccount({ variables: { verifyToken: verifyToken } })
+  }, [verifyAccount, verifyToken])
+
+  const verifyMessage = () => {
+    if (success) {
+      return (
+        <Stack isInline align="center">
+          <Icon name="check-circle" color="strong" />
+          <Text fontSize="xl">
+            <Trans>Your account email was successfully verified</Trans>
```

Comment:
Can a `home` button be put here, just to give a super duper clear way for what the user needs to do next?